langstream-runtime/langstream-runtime-impl/src/main/docker/Dockerfile

```diff
@@ -58,8 +27,8 @@ ENV NLTK_DATA="/app/nltk_data"
 
 # Install python runtime deps
 RUN cd /app && pipenv requirements --categories="packages full" > /app/requirements.txt \
-    && python3 -m pip install -r /app/requirements.txt \
-    && python3 -m nltk.downloader -d /app/nltk_data punkt averaged_perceptron_tagger
+     && python3 -m pip install -r /app/requirements.txt \
```

Comment:
since this stuff takes a lot and it's likely to never change during development, can we put it in the base image ?

Reply:
but we cannot do it

because of this line, that depends on the langstream-runtime build (the "maven" directory)

ADD maven/Pipfile.lock /app/Pipfile.lock


cc @cbornet

Reply:
I think we should isolate the python stuff in it’s own maven subproject.